File: 598-range-addition-ii/range-addition-ii.py
class Solution:
    def maxCount(self, m: int, n: int, ops: List[List[int]]) -> int:
        if not ops:
            return m*n
            
        minA = min([i[0] for i in ops])
        minB = min([i[1] for i in ops])
        return minA*minB
        

File: 598-range-addition-ii/range-addition-ii.cpp
class Solution {
public:
    int maxCount(int m, int n, vector<vector<int>>& ops) 
    {
        if (ops.empty()) return m*n;
        int ans = INT_MAX;
        int minA = INT_MAX;
        int minB = INT_MAX;

        for(auto i : ops)
        {
            minA = min(minA,i[0]);
            minB = min(minB,i[1]);
            ans = min(ans ,minA * minB);
        }
        return ans;    
    }
};